tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
 # To build with GPU support:
 $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
 
-$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
+$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
```

Comment:
You really shouldn't need sudo for this one, unless your /tmp folder isn't writable for your user, or wherever you downloaded tensorflow to isn't.